src/binding/class.rs

```diff
@@ -110,6 +117,9 @@ pub fn wrap_data<T>(klass: Value, data: T, wrapper: &dyn DataTypeWrapper<T>) ->
     unsafe { typed_data::rb_data_typed_object_wrap(klass, data, wrapper.data_type()) }
 }
 
+// TODO: Skipped the lint, but this function takes an immutable reference and returns a mutable
+// one. Changing the signature is a breaking change. What do we do?
+#[allow(clippy::mut_from_ref)]
```

Comment:
@goyox86 For breaking changes like this you should add a section to the README of migrating from Rutie 0.9 to 0.10 & Ruby 2 to Ruby 3 and about breaking changes.  The support for Ruby 3 should be Rutie release 0.10.0.